recipes/iptkl/meta.yaml

```diff
@@ -0,0 +1,86 @@
+{% set name = "iptkl" %}
+{% set version = "0.6.4" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: "https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz"
+  sha256: 340dc6265e64a3341f0db05151edf251e1e6c8476401d2a10194fbbaeb16a7db
+
+build:
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  host:
```

Comment:
you can probably remove all of that here, but python and pip

Reply:
Sure, thanks a lot for your feedback. I have removed all the libraries in the host section, except for pip and python